lib/cc/engine/analyzable_files.rb

```diff
@@ -19,7 +18,7 @@ def all
       def fetch_files(paths)
         paths.map do |path|
           if path =~ %r{/$}
-            Dir.glob("#{path}/**/*.coffee")
+            Dir.glob("#{path}**/*.coffee")
```

Comment:
Right, but that also breaks down even without the extra `/` since all the files are then prepended with `./` when they regularly aren't.